### PR TITLE
feat: update TF modules

### PR DESCRIPTION
**Pull request by ArchiFleKs**

### CODE REVIEW

File: terraform/live/frontend/terragrunt.hcl [hunk 1]
```diff
@@ -62,19 +82,19 @@ inputs = {
...
-        value = local.referer
-      }
-    ]
+      domain_name            = "particule-training.s3-website.eu-west-3.amazonaws.com"
+      origin_id              = "s3-particule-training"
+      origin_path            = ""
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "http-only"
+      origin_ssl_protocols   = ["TLSv1", "TLSv1.1", "TLSv1.2"]
```
Comment (rguichard) [inline terraform/live/frontend/terragrunt.hcl:91]: is TLSv1.3 available ?

File: terraform/live/acm-particule.io/terragrunt.hcl [hunk 2]
```diff
@@ -0,0 +1,47 @@
+include {
+  path = "${find_in_parent_folders()}"
+}
+
+terraform {
+  source = "github.com/terraform-aws-modules/terraform-aws-acm?ref=v2.12.0"
+}
+
+locals {
+  aws_region  = "us-east-1"
```
Comment (rguichard) [inline terraform/live/acm-particule.io/terragrunt.hcl:10]: eu-west-3 no ?

Comment (ArchiFleKs) [reply terraform/live/acm-particule.io/terragrunt.hcl:10]: For cloudfront the region is always us-east-1 for ACM

File: terraform/live/frontend/terragrunt.hcl (hunk 1 above)
Comment (ArchiFleKs) [reply terraform/live/frontend/terragrunt.hcl:91]: Nope and : Choose the security policy that you want CloudFront to use for HTTPS connections. A security policy determines two settings: the SSL/TLS protocol that CloudFront uses to communicate with viewers, and the cipher that CloudFront uses to encrypt the content that it returns to viewers. When you use a custom SSL certificate and SNI, you must use TLSv1 or later. We recommend that you specify TLSv1.2_2019 unless your users are using browsers or devices that do not support TLSv1.2 or later. When you use a custom SSL certificate and dedicated IP addresses, we recommend that you use TLSv1. In this configuration, the TLSv1_2016, TLSv1.1_2016, TLSv1.2_2018 and TLSv1.2_2019 security policies are not available.

**Review by rguichard — APPROVED** [review]
lgtm

**Pull request merged**